sdk/python/feast/pyspark/abc.py

```diff
@@ -114,6 +219,10 @@ def get_arguments(self) -> List[str]:
             json.dumps(self._destination),
         ]
 
+    @property
+    def destination_path(self) -> str:
+        return self._destination["path"]
```

Comment:
nit: it is a bit inconsistent that other parameters are accessible through `get_xxx` methods and this one is a property. Maybe make it a method as well?